flopy/discretization/grid.py

```diff
@@ -275,8 +275,55 @@ def botm(self):
 
     @property
     def top_botm(self):
-        new_top = np.expand_dims(self._top, 0)
-        return np.concatenate((new_top, self._botm), axis=0)
+        elevations = None
+        if self.grid_type == "structured":
+            new_top = np.expand_dims(self._top, 0)
+            elevations = np.concatenate((new_top, self._botm), axis=0)
+        else:
+            raise ValueError(
+                "top_botm property not implemented for vertex "
+                + "and unstructured model grids"
+            )
+        return elevations
```

Comment:
```suggestion
        new_top = np.expand_dims(self._top, 0)
        return np.concatenate((new_top, self._botm), axis=0)
```

Reply:
I'd reccomend keeping this as the original code is Grid(). This works for both DIS and DISV, see DISV top comment above.

I'd reccomend adding 

```
@property
def top_botm(self):
    new_top = np.expand_dims(self._top, 0)
    new_botm = np.expand_dims(self._botm, 0)
    return np.concatenate((new_top, new_botm), axis=0)
```

to the `UnstructuredGrid` class to override base class functionality.